main.go

```diff
@@ -62,14 +62,20 @@ func main() {
 	awsConfig := aws.NewConfig().WithRegion(config.RDSConfig.Region)
 	awsSession := session.New(awsConfig)
 
-	iamsvc := iam.New(awsSession)
 	rdssvc := rds.New(awsSession)
-	dbInstance := awsrds.NewRDSDBInstance(config.RDSConfig.Region, iamsvc, rdssvc, logger)
+	stssvc := sts.New(awsSession)
+
+	dbInstance := awsrds.NewRDSDBInstance(config.RDSConfig.Region, rdssvc, stssvc, logger)
 
 	sqlProvider := sqlengine.NewProviderService(logger)
 
 	serviceBroker := rdsbroker.New(config.RDSConfig, dbInstance, sqlProvider, logger)
 
+	err = serviceBroker.CheckAndRotateCredentials()
```

Comment:
I think this should run in a separate goroutine so that it doesn't delay startup of the broker. As it stands, startup will take an ever increasing amount of time as the number of RDS instances grows.